apps/server/src/shared/domain/domainobject/user.do.ts

```diff
@@ -35,6 +35,10 @@ export class UserDO extends BaseDO {
 
 	preferences?: Record<string, unknown>;
 
+	lastLoginSystemChange?: Date;
+
+	outdated?: Date;
```

Comment:
What exactly is the purpose of this property? Does it define, since when the user is outdated? (Maybe we should choose another name like `outdatedAt` or `outdatedSince`?)

Reply:
oudatedSince it is :) thx